android/build.gradle

```diff
@@ -127,5 +132,5 @@ dependencies {
   // noinspection GradleDynamicVersion
   api 'com.facebook.react:react-native:+'
   implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
-  implementation "androidx.core:core-ktx:$kotlin_version"
+  implementation "androidx.core:core-ktx:1.7.0"
```

Comment:
@braandl why is it good idea to move from a variable to hardcoded version?

Reply:
It's not. I think this slipped through, I am sorry.